### ISSUE TITLE
Mismatched number of arguments     Quick_start problem

### ISSUE THREAD

**Issue by wonderai** [zhan-xu/RigNet#17]
`loading all networks...
     joint prediction network loaded.
     root prediction network loaded.
     connection prediction network loaded.
     skinning prediction network loaded.
creating data for model ID 17872
     gathering topological edges.
     calculating surface geodesic matrix.
surface geodesic calculation: 6.194427251815796 seconds
     gathering geodesic edges.
predicting joints
predicting connectivity
Traceback (most recent call last):
  File "/home/wonder/Documents/RigNet/quick_start.py", line 434, in <module>
    mesh_filename=mesh_filename.replace("_remesh.obj", "_normalized.obj"))
  File "/home/wonder/Documents/RigNet/quick_start.py", line 183, in predict_skeleton
    root_id = getInitId(input_data, root_pred_net)
  File "/home/wonder/Documents/RigNet/mst_generate.py", line 105, in getInitId
    root_prob, _ = model(data, shuffle=False)
  File "/home/wonder/anaconda3/envs/rigrig/lib/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/wonder/Documents/RigNet/models/ROOT_GCN.py", line 140, in forward
    joint_feature = self.joint_encoder(torch.abs(joints_norepeat[:, 0:1]), joints_norepeat, joints_batch)
  File "/home/wonder/anaconda3/envs/rigrig/lib/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/wonder/Documents/RigNet/models/ROOT_GCN.py", line 98, in forward
    sa1_joint = self.sa1_joint(*sa0_joint)
  File "/home/wonder/anaconda3/envs/rigrig/lib/python3.6/site-packages/torch/nn/modules/module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/wonder/Documents/RigNet/models/ROOT_GCN.py", line 25, in forward
    max_num_neighbors=64)
  File "/home/wonder/anaconda3/envs/rigrig/lib/python3.6/site-packages/torch_geometric/nn/pool/__init__.py", line 172, in radius
    num_workers)
TypeError: radius() takes from 3 to 6 positional arguments but 7 were given

Process finished with exit code 1
`

Hi I am trying to run the demo quick_start, but I met the problem shown above, mismatched number of arguments, can you please assist.

**Comment by wonderai** [comment]
mismatched version of torch-geometric

**Comment by zhan-xu** [comment]
ok, which version were you using? I am testing under latest version but didn't see this problem.

**Comment by wonderai** [comment]
I was using the latest version with CUDA 11.0 that I couldt install your required pytorch-sparse version. So I followed the install method provided by pytorch-sparse author     pip install torch-scatter==latest+${CUDA} torch-sparse==latest+${CUDA} -f https://pytorch-geometric.com/whl/torch-1.4.0.html , so did torch-scatter, torch-cluster and torch-spline-conv. Then the error raised. The version might be pytorch-sparse 0.6.7   torch-scatter 2.0.5  torch-cluster 1.5.7 torch-spline-conv 1.2.0
After I downgraded my cuda to 10.2, I successfully installed your required version of libraries and ran out the result.
